app/frontend/src/routes.ts

```diff
@@ -16,8 +16,9 @@ export const baseRoute = "/";
 export const couchersURL = "https://couchers.org";
 export const forumURL = "https://community.couchers.org";
 
-export const contributeRoute = "/contribute";
-export const donationsRoute = "/donate";
+export const donationsRoute = "/donatemore";
+export const donationRoute = "/donate";
+export const donationsMixedRoute = "/donatemixed";
```

Comment:
Why do we need three routes here? Shouldn't `/donate` be enough since this is all on one page?

Reply:
He mentioned to me that this is just temporary while its WIP